.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           environment:
           command: |
             export PATH=$HOME/conda/envs/glow/bin:$PATH
-            sbt printTests coverage "testOnly $(circleci tests split --split-by=timings --timings-type=classname core-test-names.log | tr '\n' ' ') --" coverageReport exit
+            sbt printTests coverage "core/testOnly $(circleci tests split --split-by=timings --timings-type=classname core-test-names.log | tr '\n' ' ') --" coverageReport exit
```

Comment:
This makes sense to me, but it is this change necessary for this PR? I'm fine with adding it now, just trying to understand.

Reply:
Ah whoops, I forgot to change this back after I set `root`.